### ISSUE TITLE
Feature request : Key/Value format Json button

### ISSUE THREAD

**Issue by charithsoori** [ThreeMammals/Ocelot#326]
When the user clicks, a button JSON will be formatted to improve readability.

Requirement.
I recently enabled hosting OcelotConfiguration file in consul Consul  using following code
```
 services.AddOcelot(Configuration)
                .AddStoreOcelotConfigurationInConsul();
```
Generated keyvaue node under OcelotConfiguration is not formatted in UI.

**Comment by TomPallister** [comment]
@charithsoori thanks for your interest in the project!

I cannot add a button but I can change Ocelot so it always stores the JSON indented and this will achieve what you want.

**Comment by TomPallister** [comment]
will close after next release

**Comment by TomPallister** [comment]
released in 5.5.6
